specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,1111 @@
+            "SetBlobTags",
+            "SetBlobImmutabilityPolicy",
+            "SetBlobLegalHold",
+            "SetBlobExpiry",
+            "DeleteBlob",
+            "UndeleteBlob"
+          ],
+          "x-ms-enum": {
+            "name": "StorageTaskOperationName",
+            "modelAsString": true
+          }
+        },
+        "parameters": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Key-value parameters for the operation."
+        },
+        "onSuccess": {
+          "type": "string",
+          "description": "Action to be taken when the operation is successful for a object.",
+          "enum": [
+            "continue"
+          ],
+          "x-ms-enum": {
+            "name": "onSuccess",
+            "modelAsString": false
+          }
+        },
+        "onFailure": {
+          "type": "string",
+          "description": "Action to be taken when the operation fails for a object.",
+          "enum": [
+            "break"
+          ],
+          "x-ms-enum": {
+            "name": "onFailure",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Represents an operation to be performed on the object"
+    },
+    "StorageTasksListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTask"
+          },
+          "x-ms-identifiers": [],
+          "description": "Gets the list of storage tasks and their properties."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of storage tasks. Returned when total number of requested storage tasks exceed maximum page size.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Storage Tasks operation."
+    },
+    "StorageTaskAssignmentsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskAssignment"
+          },
+          "x-ms-identifiers": [],
+          "description": "Gets the list of storage task assignment Ids."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of storage task assignment Ids. Returned when total number of requested storage task assignment Ids exceed maximum page size.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Storage Tasks operation."
+    },
+    "StorageTaskAssignment": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "arm-id",
```

Comment:
>"format": "arm-id", [](http://example.com/codeflow?start=10&length=19)

Also, add this:

"allowedResources": [
          {
            "type": "Microsoft.StorageTasks/storageTasks/storageTaskAssignments"
          }
        ]

Reply:
Updated the resource type constraint

Reply:
Here, it is till not updated.